index.rst

```diff
@@ -1,83 +1,144 @@
 
-  Subsections
-  -----------
+.. figure:: /_static/Dipoles_ds9_fourpanel_f012.png
+   :name: fig-dipole_fourpanel
+   :target: https://github.com/lsst-dm/dmtn-019/tree/master/_static/Dipoles_ds9_fourpanel_f012.png
 
-  and
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+   A small region from the "Faint" simulated images at airmass 1.3 differenced from a template at airmass 1.00. Left to right, top to bottom: u-band, g-band, r-band, and the g-band template image for reference.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
-     :target: http://target.link/url
+.. figure:: /_static/Dipoles_ds9_fourpanel_f012_test8.png
+   :name: fig-dipole_fourpanel8
+   :target: https://github.com/lsst-dm/dmtn-019/tree/master/_static/Dipoles_ds9_fourpanel_f012_test8.png
 
-     Caption text.
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-report-bootstrap or
-   this repo's README for more info.
+   As Figure 1, but for the "Bright" simulated images which contain four times as many bright stars.
 
-   Feel free to delete this instructional comment.
 
-:tocdepth: 1
+At first glance, it is apparent that dipoles are more prevalent in the g-band difference image of both simulations, which is due to three factors: 
 
-In order to test Differential Chromatic Refraction (DCR) correction algorithms we first need a metric that quantifies the severity of residual dipoles in difference images, in order to evaluate the performance of the correction. 
-I used the StarFast simulator (http://dmtn-012.lsst.io, https://github.com/lsst-dm/starfast_simulator) to generate multiple realizations of a small field at u, g, and r bands and across a range of airmasses from 1.0 to 2.0.
-I then used ingestSimImages.py and processEimage.py from obs_lsstSim to generate standard calibrated exposures, and ran standard LSST image differencing using imageDifference.py with a calibrated exposure near zenith used as the template (using GetCalexpAsTemplateTask).
+#. The g-band filter is twice as wide in wavelength as u-band, balancing the larger change in index of refraction at u-band (see Figure 1 of https://github.com/isullivan/LSST-DCR/blob/master/DCR%20literature%20overview.pdf)
+#. Most stars of type A or cooler have a uniformly rising spectrum in u-band, which results in a similarly dispersed psf (see the dipole images of individual stars at the bottom of https://github.com/lsst-dm/starfast_simulator/blob/master/StarFast_example.ipynb). In g-band, by contrast, a typical type A star will have a falling spectrum, a type K star will have a rising spectrum, and in between stars will have a spectrum that is on average flat, but full of features. As a result, there is a greater variation in PSF shape due to DCR in g-band than in any other band.
```

Comment:
Might be worthwhile quantifying this effect -- which stellar types are most different from which other stellar types (and vice versa) -- possible helping us to choose stellar types/colors to standardize on. (Possibly not for this technote. Just a thought.)